packages/ckeditor5-dev-tests/bin/testmanual.js

```diff
@@ -9,7 +9,7 @@ import chalk from 'chalk';
 import path from 'path';
 import * as tests from '../lib/index.js';
 
-const options = tests.parseArguments( process.argv.slice( 2 ) );
+const options = tests.parseArguments( process.argv.slice( 2 ), { allowDefaultIdentityFile: true } );
```

Comment:
Is there a specific reason to introduce a second parameter?

Couldn't we extend the `minimistConfig` in `parseArguments` itself to add this flag with some default value?